crates/bevy_winit/src/winit_monitors.rs

```diff
@@ -0,0 +1,12 @@
+use winit::monitor::MonitorHandle;
+
+use bevy_ecs::entity::Entity;
+use bevy_ecs::system::Resource;
+
+/// Stores [`winit`] monitors and their corresponding entities
+#[derive(Resource, Debug, Default)]
+pub struct WinitMonitors {
+    /// Stores [`winit`] monitors and their corresponding entities
+    // we can't use a `HashMap` here because `MonitorHandle` doesn't implement `Hash` :(
```

Comment:
Looked into it a bit and saw that this might not be a PR that's too difficult as it already implements `Ord` for example, but it's gonna have to compile on all platforms. You could go for a BTreeMap as well.

Maybe add a convenience method for this for getting an entry by index or handle to make it slightly nicer in `winit_window_position`? Maybe you can also get away with this only being `pub(crate)`.

Reply:
Unfortunately `pub(crate)` is kind of infectious and results in making basically all of `bevy_winit` only crate visible. Which probably wouldn't be a bad thing but is technically a breaking change. I'm curious if anyone actually depends directly on this crate.

Reply:
Oh I meant only the field could be `pub(crate)` (or not `pub` at all) to hide the implementation details.